cafebabel/templates/articles/proposals/create.html

```diff
@@ -57,6 +57,20 @@ <h3>Wanna be part of Cafébabel project? That’s the right place</h3>
 
     <button type=submit>Suggest this article</button>
   </form>
-  <button><a href=m&#x61;ilto:{{config.EDITORS_EMAIL_DEFAULT|urlencode|replace('.', '&#46;')|replace('a', '&#x61;')|safe}}>Or send your proposal by email</a></button>
+  <a class="button proposal-email" href=m&#x61;ilto:{{config.EDITORS_EMAIL_DEFAULT|urlencode|replace('.', '&#46;')|replace('a', '&#x61;')|safe}}>Or send your proposal by email</a>
+  <script>
+     const editorEmail = {
+         en: 'editor@cafebabel.com',
```

Comment:
Do these email actually exist? If so we cannot merge as it will be email robot food. We should make a minimalistic tweak not to be so easy to read or replace with a fake email in this code while we're still not reading from the config file.

Reply:
ok, how to do that ? @vinyll

Reply:
I'll amend this PR with the emails from the settings

Reply:
what do you mean ?

Reply:
I meant [I had to do this](https://github.com/cafebabel/cafebabel.com/pull/113/commits/eb96788de7f67757bc35bfdb10217bcdd641854e#diff-269b2c3dee5f415b4d0b8c2aedbc5da8R62)